src/main/java/seedu/momentum/logic/SettingsUpdateManager.java

```diff
@@ -25,15 +35,33 @@ public static void initSettingsUpdateManager(Ui appUi, StatisticGenerator appSta
      * Updates the theme of the application.
      */
     public static void updateTheme(Theme theme) {
-        ui.getMainWindow().updateTheme(theme);
+        Optional.ofNullable(ui).ifPresentOrElse(ui -> ui.getMainWindow().updateTheme(theme), logUiNotPresent());
```

Comment:
Can UI or Statistic ever be null? I think it might be better to make sure that they're never null using requirenonnull rather try to handle null values. If either are ever null, there would be bug in other parts of the program anyways.

Reply:
Yes, but in tests only, particularly these:
![Screenshot 2020-11-04 at 9 31 06 PM](https://user-images.githubusercontent.com/67047571/98117656-0e3aa300-1ee5-11eb-88ea-e059d5d0d742.jpg)
All of these tests invoke setTrackedItem() from Model, which will call resetUi(), which then calls updateApplicationSettings() in SettingsUpdateManager. However, SettingsUpdateManager is not initialised in the test and so Ui and Statistics will be null, leading to NullPointerException when running the tests.

Reply:
Is there any followup to this??

Reply:
I think its a small issue, so can just merge as it is